toc/working-groups/app-runtime-interfaces.md

```diff
@@ -67,91 +67,219 @@ areas:
   - cloudfoundry/app-autoscaler-env-bbl-state
   - cloudfoundry/app-runtime-interfaces-infrastructure
 
-- name: Buildpacks
+- name: Buildpacks Content
```

Comment:
Wouldn't `Buildpacks Docs` be a more descriptive name?

Reply:
We based these teams off the existing [Paketo teams](https://github.com/orgs/paketo-buildpacks/teams). Happy to change them if we want; we just opted for the same names for consistency and to reduce confusion.

Reply:
Also, to provide more context on the naming of the equivalent Paketo group - `Content` was chosen (rather than docs) because on the Paketo side that sub group is responsible for additional content beyond documentation - they maintain the website and blog as well.

Given we don't have that equivalent for CF buildpacks, I'm happy to call it `Docs` if you prefer.